index.html

```diff
@@ -0,0 +1,738 @@
+                            </div>
+                        </div>
+
+        
+
+
+            <div class="latest-news-thumbnail">
+                <div class="thumbnail-img">
+                    <img src="images/mapping-apps.png" alt="start mind mapping apps">
+                </div>
+                <div class="thumbnail-text">
+                    <h3 class="thumbnail-title"><a href="#">How to get started with mind mapping apps</a></h3>
+                    <span class="author-detail"><a href="#">Matthew Beedham</a> 7 hours ago</span>
+                </div>
+            </div>
+
+            <div class="latest-news-thumbnail">
+                <div class="thumbnail-img">
+                    <img src="images/speaker-sweet-deal.png" alt="speaker sweet deal">
+                </div>
+                <div class="thumbnail-text">
+                    <h3 class="thumbnail-title"><a href="#">CHEAP: $270 off Harman Kardon's Onyx 4 speaker sounds like a sweet deal</a></h3>
+                    <span class="author-detail"><a href="#">TNW Deals</a> 6 hours ago</span>
+                </div>
+            </div>
+
+            <div class="latest-news-thumbnail">
+                <div class="thumbnail-img">
+                    <img src="images/youtube-parents-love.png" alt="parents love youtube">
+                </div>
+                <div class="thumbnail-text">
+                    <h3 class="thumbnail-title"><a href="#">How YouTube became the app parents love to hate</a></h3>
+                    <span class="author-detail"><a href="#">Ravie Lakshmanan</a> 7 hours ago</span>
+                </div>
+            </div>
+            <div class="latest-news-thumbnail">
+                <div class="thumbnail-img">
+                    <img src="images/3d-spintronics.png" alt="3D hard drive performance">
+                </div>
+                <div class="thumbnail-text">
+                    <h3 class="thumbnail-title"><a href="#">How 3D spintronics will take hard drive performance to the next level</a></h3>
+                    <span class="author-detail"><a href="#">The Conversation</a> 7 hours ago</span>
+                </div>
+            </div>
+            
+
+        </div>
+
+
+        <!--LATEST FUNDING-->
+
+        <div class="latest-funding">
+            <h3 class="latest-funding-title"><a href="#">Latest funding rounds on Index.co <i class="fas fa-greater-than"></i></a></h3>
+            <ul>
+                <li><a href="#">Add info | </a></li>
+                <li><a href="#">Get updates by email | </a></li>
+                <li><a href="#">See more</a></li>
+            </ul>
+            <span><a href="#" class="span1 info">Add info </a> <a href="#" class="span1 info">Get updates by email </a> <a href="#" class="span1">See more</a></span>
+        </div>
+
+
+        <div class="latest-funding-details">
+                <div class="subjectwell">
```

Comment:
This should be an article tag instead. Each one of the 3 latest funding articles must be inside an article tag. Kindly do that with them all.